### ISSUE TITLE
Mushu shop required tenant admin in order to implement - due to policy.tf

### ISSUE THREAD

**Issue by adizohar** [oracle-quickstart/oci-cloudnative#189]
Hi Team

The shop requires admin in order to implement the policy in policy.tf
Please do amend the code to create the policy under the compartment

 ```
# Create policy to allow use lifecycle
resource "oci_identity_policy" "mushop_allow_object_storage_lifecycle" {
  name           = "mushop-object-family-${random_id.mushop_id.dec}"
  description    = "policy created by terraform for MuShop Basic"
  compartment_id = var.compartment_ocid
  statements     = ["Allow service objectstorage-${var.region} to manage object-family in compartment id ${var.compartment_ocid}"]
  freeform_tags  = local.common_tags
}
```

**Comment by junior** [comment]
Was a requirement of the mushop **basic** to use the tenancy root to help simplify the deployment for new users and users that are not yet familiar with the OCI concepts of compartments.

For a better advanced experience I suggest to try the mushop complete [v2.0.0](https://github.com/oracle-quickstart/oci-cloudnative/releases/tag/v2.0.0).

There are already plans to add advanced options for the mushop basic, which will happen with the v1.2.0 of the MuShop basic. Will have options to select shapes, compartments and ignore policies. Will be a hidden pane on the schema. This will also affect the Solution Hub e-commerce sample.

**Comment by adizohar** [comment]
Hi Junior, 

I checked the basic script, it does require customer to provide customer_ocid in order to deploy the shop inside the compartment. 
If the user has only permission to the specific compartment it will fail on the policy

Simple fix will allow user to deploy the basic shop inside compartment as well without admin access, it will work also on admin with root compartment as well

Just trying to help customers with access to compartment only.

For your info

**Comment by jeevanjoseph** [comment]
@adizohar thanks for the suggestion  - we are looking at adding this to the next release that will add some advanced options to give users better control over the deployment.

That being said, Out of curiosity, why you are providing `user_ocid` - are you running this as a terraform config directly against your tenancy ?

While that is certainly supported, the primary usecase to for people to download the resource manager stack and import that- which saves them the trouble of having to learn or understand terraform.

**Comment by adizohar** [comment]
@jeevanjoseph , I am not following regards user_ocid ? I didn't specify user_ocid, I specified compartment_ocid

Anyway, I will close this issue

**Comment by jeevanjoseph** [comment]
@adizohar - are you cloning the repo and setting up terra form to run the terraform code from your laptop/workstation ?

Or are you using the Oracle Resource Manager  feature?See MuShop basic instructions for new users here : https://github.com/oracle-quickstart/oci-cloudnative/blob/master/README.md

**Comment by junior** [comment]
enhancement included on the PR #192
